src/Text/Pandoc/Writers/Muse.hs

```diff
@@ -276,6 +276,7 @@ blockToMuse (Table _ blkCapt specs thead tbody tfoot) =
     isSimple = onlySimpleTableCells (headers : rows) && all (== 0) widths
 blockToMuse (Div _ bs) = flatBlockListToMuse bs
 blockToMuse Null = return empty
+blockToMuse (Figure {}) = return empty
```

Comment:
This old behaviour slipped through the cracks from the phase where `Figure` elements were simply ignored.
This should now change to the phase were `Figure` is treated as a `Div` as a default handling strategy:

```{haskell}
blockToMusc (Figure attr _ bs) = blockToMuse (Div attr bs)
```